mw-webapp/src/component/button/Button.tsx

```diff
@@ -13,7 +13,7 @@ export interface ButtonProps {
   /**
    * Callback triggered on button click
    */
-  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
```

Comment:
why?

Reply:
PLease don't use onClick optionally. If you have a case where you don't need onClick you can use empty callback

Reply:
OK) I did it